src/etc/inc/plugins.inc.d/openvpn.inc

```diff
@@ -31,6 +31,8 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
+require_once("interfaces.lib.inc");
```

Comment:
```suggestion
```

includes for plugins go to calling pages, not the plugin itself.. likely openvpn page already loads interfaces.inc... if not it should now